src/html/parse-markdown.js

```diff
@@ -39,9 +39,6 @@ const thbreak = {
 function parse({ content: { body = '' } = {} }, { logger }) {
   logger.debug(`Parsing markdown from request body starting with ${body.split('\n')[0]}`);
 
-  // disable setext headings (=== and ---)
-  remark.Parser.prototype.blockTokenizers.setextHeading = () => {};
-
   const preprocessor = unified()
     .use(remark, { setext: false })
```

Comment:
Does this `{ setext: false }` flag do anything?

Reply:
I was wondering the same thing actually. Our tests pass with and without this option, but I wasn't entirely certein and wanted to keep my changes as surgical as possible. `remark` [docs](https://github.com/remarkjs/remark/tree/master/packages/remark-stringify#optionssetext) suggest it does enable or disable setext headings, though...

Reply:
That appears to be a `remark-stringify` option :) I think we can remove it.